nav2_bringup/bringup/package.xml

```diff
@@ -26,6 +26,7 @@
   <test_depend>ament_cmake_pytest</test_depend>
   <test_depend>launch</test_depend>
   <test_depend>launch_testing</test_depend>
+  <test_depend>python-zmq</test_depend>
```

Comment:
I think this needs to be a depend

Reply:
Also if it is only needed for testing? This is only needed for the tests with zmq. The zmq inside the BT.CPP is a cpp and b solved by BT.CPP directly

Reply:
It is not only neededfor testing. If you set `enable_groot_monitoring` to true, then its used at runtime.

Reply:
Oh shoot. Yes, you are right. It calls the bringup stuff, but this `depend` needs to live in `system_tests`.
And there just as `<test_depend>`, as python-zmq's only occasion lives inside a `gtest`. Now we should have it solved.. :+1: